### PR TITLE
Security & Hardening Guide: New chapter about restricting cron and at

### PR DESCRIPTION
**Pull request by cwickert**
### PR creator: Description

This PR migrates two chapters from the SAP HANA Hardening Guide to the SLE Security & Hardening Guide.

### PR creator: Are there any relevant issues/feature requests?

* [BSC#1176613: Migrate chapter 'Implement cron.allow' from OS Hardening Guide for SAP HANA to SLES Hardening Guide](https://bugzilla.suse.com/show_bug.cgi?id=1176613)
* [BSC#1176614: Migrate chapter 'Implement at.allow' from OS Hardening Guide for SAP HANA to SLES Hardening Guide](https://bugzilla.suse.com/show_bug.cgi?id=1176614)

### PR creator: Which product versions do the changes apply to?

When opening a PR, check all versions of the documentation that your PR applies to.

- SLE 15/openSUSE Leap 15.x
  - [x] SLE 15 SP4/openSUSE Leap 15.4 *(current `main`, no backport necessary)*
  - [x] SLE 15 SP3/openSUSE Leap 15.3
  - [x] SLE 15 SP2/openSUSE Leap 15.2
  - [ ] SLE 15 SP1
  - [ ] SLE 15 GA
- SLE 12
  - [ ] SLE 12 SP5
  - [ ] SLE 12 SP4
  - [ ] SLE 12 SP3

### PR reviewer only: Have all backports been applied?

The doc team member merging your PR will take care of backporting to older documents.
When opening a PR, do *not* set the following check box.

- [ ] all necessary backports are done

### CODE REVIEW

File: xml/security_cron_at.xml [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+    This chapter explains how to restrict access to the <systemitem
+     class="daemon">cron</systemitem> and <systemitem
+     class="daemon">at</systemitem> daemons to improve the security of a system.
+   </para>
+  </abstract>
+ </info>
+ <sect1>
+  <title>Restricting the <systemitem class="daemon">cron</systemitem> daemon</title>
+  <para>
+   The Linux <systemitem class="daemon">cron</systemitem> system is used to
```
Comment (jsegitz) [inline xml/security_cron_at.xml:25]: -Linux, cron is available for many systems

File: xml/security_cron_at.xml [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+    </para>
+<screen>&prompt.user2;<command>at 00:00</command>
+You do not have permission to use at.</screen>
+   </step>
+  </procedure>
+   <note>
+   <title>Uninstalling <systemitem class="daemon">at</systemitem></title>
+   <para>
+    <systemitem class="daemon">at</systemitem> is not widely used anymore.
+    Consider uninstalling the daemon if you don't have valid users instead of
```
Comment (jsegitz) [inline xml/security_cron_at.xml:144]: valid use cases

**Review by jsegitz — CHANGES_REQUESTED** [review]

**Comment by cwickert** [comment]
Thanks for the review, @jsegitz. I've incorporated your feedback, please approve if you are fine with it.

**Review by jsegitz — APPROVED** [review]

**Review by cjschroder — APPROVED** [review]
Looks good, and I am glad you mentioned systemd timers.

**Comment by cwickert** [comment]
> I am glad you mentioned systemd timers.

Ah, good you mention it. I added a reference to the timer units chapter in 13b5549.

**Pull request merged**